### DISCUSSION TITLE
拡張機能の設定について

### DISCUSSION THREAD

**Discussion by hiyakake** [AyumuNekozuki/niconico-darkmode#169] [Ideas]
今は共通メニュー内にあるが
Nアニメなどないページもある
また、大百科など微妙に共通メニューが違うものもある

そう考えると、画像内のポップアップの場所にあったほうが良いのではないか

![image](https://user-images.githubusercontent.com/13294981/133915208-16c817a1-df74-49a9-80a1-11c8d26f7eee.png)

**Comment by AyumuNekozuki** [comment]
#170 @AyumuNekozuki
